checkout/frames-android
# PIMOB-2122: Improve bitrise CI flow

## Issue

PIMOB-2122

## Proposed changes

* Remove `verification-full-extra-lint` and relevant triggers as it only works for 4.0.0
* Experiment working with only `verification-full` and use `run_if` condition to run steps based on whether the PR is in a draft mode.

## Test Steps

1. Push a PR in a draft mode, verify only unit test step is run
2. Make the PR into ready for review,  verify full verification steps are run, including instrumented tests, sonar qube steps etc.

## Checklist
